src/factory.js

```diff
@@ -68,6 +69,8 @@ export class Factory {
     }
     this.renderQ = [];
     this.stave = null; // current stave
+
+    this.StaveNote = this.StaveNote.bind(this);
```

Comment:
I think this should be a local function in the test file (maybe as a wrapper called `newStaveNote`). It certainly does not belong here.